Library/OcMiscLib/Math.c

```diff
@@ -0,0 +1,60 @@
+/** @file
+ Copyright (C) 2019, vit9696. All rights reserved.
+
+ All rights reserved.
+
+ This program and the accompanying materials
+ are licensed and made available under the terms and conditions of the BSD License
+ which accompanies this distribution.  The full text of the license may be found at
+ http://opensource.org/licenses/bsd-license.php
+
+ THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+ **/
+
+#include <Library/BaseLib.h>
+#include <Library/OcMiscLib.h>
+
+/**
+ MultThenDivU64x64x32 is from MdeModulePkg's PciRootBridgeIo.c
+
+ Copyright (c) 1999 - 2018, Intel Corporation. All rights reserved.<BR>
+ SPDX-License-Identifier: BSD-2-Clause-Patent
+ **/
+UINT64
+MultThenDivU64x64x32 (
+                      IN      UINT64                    Multiplicand,
```

Comment:
I get a feeling something got borked with the spacing here, could make it more usual?
Two spaces till IN/OUT, then up to 1 to type, and up to 2 to arguments… Like

```C
VOID
DataHubLogData (
  IN  EFI_GUID  *DataRecordGuid,
  IN  EFI_GUID  *ProducerName,
  IN  VOID      *RawData,
  IN  UINT32    RawDataSize
  );
```

Reply:
Fixed in latest.